src/main/java/core/basesyntax/Parameters.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Parameters {
+    private static final int FIGURE_COUNT = 5;
+    private static final double DEFAULT_RADIUS = 10.0;
+
+    public static int getFigureCount() {
+        return FIGURE_COUNT;
+    }
+
+    public static double getDefaultRadius() {
+        return DEFAULT_RADIUS;
+    }
+}
```

Comment:
```suggestion
    public static final int FIGURE_COUNT = 5;
    public static final double DEFAULT_RADIUS = 10.0;
}
```
this is the usual pattern for class of constants